### ISSUE TITLE
mode="push" position="right" does not work as expected

### ISSUE THREAD

**Issue by epetrashen** [arkon/ng-sidebar#98]
Thanks for the component! It's really good, a lot of inputs and outputs. However there's this thing I struggle with & suspect that might be a bug.
When I use `mode="push" position="right"` the sidebar doesn't actually "push" the content. The same sidebar does push it when I remove `position="right"` property.  Is there maybe a known workaround for this?
![ng-sidebar](https://user-images.githubusercontent.com/1194010/27539195-3720441a-5a84-11e7-84ff-e0c79076a0a5.png)

**Comment by epetrashen** [comment]
sorry, this seems to be an issue related to our styles.

**Comment by arkon** [comment]
It's probably worth looking at [the demo](https://echeung.me/ng-sidebar/) for what's meant to be expected behaviour.

Currently, `push` mode reflows the content, while `slide` actually slides the entire content over, so that might be what you're looking for instead.

**Comment by epetrashen** [comment]
@arkon thanks so much for your help! indeed `slide` seems like exactly what we are looking for. However, when I use it with showBackdrop="true" I get backdrop covering the whole screen, including a sidebar?

And btw https://echeung.me doesn't work for me, each time I try to access it (both yesterday and today) I get _This site can’t be reached_ message in a browser.

**Comment by arkon** [comment]
@epetrashen Yeah, the backdrop thing was a bug. It's fixed in v6.0.0. The site loads fine for me. It might be blocked by your workplace?

**Comment by epetrashen** [comment]
@arkon You're right, I was actually able to open it from my cellphone, so it's probably blocked by the workplace/provider. 
I'm sorry to be a pest, but I tried to update to v6.0.0 to get those beautiful `slide` mode sidebars and I now just get blank space everywhere inside `ng-sidebar-container` - in both actual project and in a dummy project without any styles I scraped together just for testing. Is that a known issue maybe? 😣
